addons/pos_sale/report/sale_report.py

```diff
@@ -74,7 +76,9 @@ def _query(self, with_clause='', fields={}, groupby='', from_clause=''):
                     LEFT JOIN pos_config config ON (config.id = session.config_id)
                 left join product_pricelist pp on (pos.pricelist_id = pp.id)
         '''
+        return from_
 
+    def _group_by_pos(self, groupby=''):
```

Comment:
```suggestion
    def _group_by_pos(self):
```
Same comment than for `_from_pos`